docs/plugins/tv.md

```diff
@@ -1,35 +1,35 @@
 # TV
 
-The TV plugin serves as an abstraction layer to the _live TV_ functionality of a STB. The interface gives you access to the current channel
-and program information, and allows you to change the TV channel from inside an App (if your app is _whitelisted_ to do so).
+The *TV* plugin serves as an abstraction layer to the *live TV* functionality of an STB. The interface gives access to the current channel
+and program information, and allows you to change the TV channel from *inside* an App (if your App is *whitelisted* for that purpose).
 
-With the TV plugin you can adapt your App _contextually_ to what the user is currently watching on TV.
+You can use the TV plugin to adapt your App contextually to what the user is currently watching on TV.
```

Comment:
This wording makes it sounds as if it's the _only_ thing you can do with the plugin. But it's _a_ thing you can do. The previous sentence maybe also doesn't cover that completely, by the way.

Reply:
Agreed. Changed the sentence into "One of the things you can do with the TV plugin, is adapt your App contextually to what the user is currently watching on TV."